src/WebJobs.Script.WebHost/Models/RunFromPackageContext.cs

```diff
@@ -46,6 +46,11 @@ public async Task<bool> BlobExistsAsync(ILogger logger)
                 }
             }, maxRetries: 2, retryInterval: TimeSpan.FromSeconds(0.3));
 
+            if (!exists)
+            {
+                logger.LogWarning($"{EnvironmentSettingNames.ScmRunFromPackage} points to an empty location. Function app has no content.");
```

Comment:
Replace {EnvironmentSettingNames.ScmRunFromPackage} with {EnvironmentVariableName} in case this method gets called for other scenarios later.